Documentation/docs-site/content/tutorial/access-secret-cfgmap-in-function.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Access secret/configmap in function"
+draft: false
+weight: 41
+---
+
+From fission v0.5.0 and later, functions are able to access [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) and [ConfigMaps](https://kubernetes.io/docs/concepts/storage/volumes/#configmap) specified by users.
+
+### Create Secret and ConfigMap
+
+You can create Secret and ConfigMap with CLI.
+
+``` bash
+$ kubectl -n default create secret generic foo --from-literal=TEST_KEY="TESTVALUE"
+$ kubectl -n default create configmap bar --from-literal=TEST_KEY=TESTVALUE
+```
+
+Or use `kubectl create -f <filename.yaml>` to create these from a YAML file.
+
+``` yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: default
+  name: foo
+data:
+  TEST_KEY: VEVTVFZBTFVF # value after base64 encode
+type: Opaque
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: default
+  name: bar
+data:
+  TEST_KEY: TESTVALUE
+```
+
+### Access Secret and ConfigMap
+
+** NOTICE **: Due to current design constraints, the value of Secret and ConfigMap that functions access to may not consistent with the latest value. For now, you can recreate the function to update the value of Secret and ConfigMap.
```

Comment:
1. Suggest replacing `** NOTICE **: ` with simply `Note: `.
2. Let's move the note to the end

Also let's make it more specific:

Note: If the Secret or ConfigMap value is updated, the function may not get the updated value for some time; it may get a cached older value.